frontend/src/component/feature/FeatureView/FeatureOverview/FeatureLifecycle/FeatureLifecycleTooltip.test.tsx

```diff
@@ -23,24 +22,17 @@ const renderOpenTooltip = (
     loading = false,
 ) => {
     render(
-        <Routes>
-            <Route
-                path={'/projects/:projectId'}
-                element={
-                    <FeatureLifecycleTooltip
-                        stage={stage}
-                        onArchive={onArchive}
-                        onComplete={onComplete}
-                        onUncomplete={onUncomplete}
-                        loading={loading}
-                    >
-                        <span>child</span>
-                    </FeatureLifecycleTooltip>
-                }
-            />
-        </Routes>,
+        <FeatureLifecycleTooltip
+            stage={stage}
+            onArchive={onArchive}
+            onComplete={onComplete}
+            onUncomplete={onUncomplete}
+            loading={loading}
+            project={'default'}
+        >
+            <span>child</span>
```

Comment:
curious: what is this child? is it just the text child in a span?

Reply:
this component accepts any ReactElement as children prop so wanted to specify sth that doesn't look like string. In tests we find this element by text so passing child without a span would also work